test_tasks/Task_5.py

```diff
@@ -0,0 +1,12 @@
+# 5.
+def is_palindrome(input_value):
+    input_str = str(input_value)
+    return input_str == input_str[::-1]
```

Comment:
`input` returns `str`, so no need to convert
```suggestion
    return input_value == input_value[::-1]
```